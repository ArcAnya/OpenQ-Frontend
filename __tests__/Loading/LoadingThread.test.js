import React from 'react';
import LoadingThread from '../../components/Loading/LoadingThread';
import renderer from 'react-test-renderer';

describe('LoadingThread', () => {
  it('should match DOM Snapshot', () => {
    const tree = renderer.create(<LoadingThread changeText={false} />);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
