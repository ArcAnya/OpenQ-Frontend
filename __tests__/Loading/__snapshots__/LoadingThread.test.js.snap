// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingThread should match DOM Snapshot 1`] = `
<div
  className="flex fixed top-0  bg-link-colour
	   animationDefault
	   false
	   animationStart
	   false 
	 
	  false
	  z-40 py-px bg-link-color self-start left-0 w-full bg- border-web-gray h-0.5 "
/>
`;
